src/Platforms/PostgreSQLPlatform.php

```diff
@@ -1186,6 +1186,21 @@ public function getTruncateTableSQL($tableName, $cascade = false)
         return $sql;
     }
 
+    /**
+     * Get the snippet used to retrieve the default value for a given column
+     *
+     * @return string
+     */
+    public function getDefaultColumnValueSQLSnippet()
```

Comment:
```suggestion
     */
    public function getDefaultColumnValueSQLSnippet(): string
```

Reply:
that's the only methods with type hinting ? but ok

Reply:
Yes, because most of them come from a time when PHP didn't have return types. Adding native types to a class that is supposed to be extended in userland is a breaking change. But that should not stop us from using native return types on brand new methods.

Reply:
fair enough, I've applied your suggestion and corrected the remarks.